<launch> 
  <param name="lidar_info_path" type="string" value="/media/zz/new/myMidImg/kitti360_00/out.txt" />

  <param name="xy_leaf_size" type="double" value="0.5" />
  <param name="z_leaf_size" type="double" value="0.2" />

  <param name="landmark_occupancy_threshold" type="int" value="4" />
  <param name="landmark_range_threshold" type="double" value="40" />
  <param name="landmark_mask_radius" type="double" value="3" />

  <param name="lsh_band_length" type="int" value="6" />
  <param name="lsh_band_num" type="int" value="10" />

  <param name="occupancy_context_max_range" type="double" value="20" />
  <param name="occupancy_context_redius_resolution" type="double" value="0.5" />
  <param name="occupancy_context_angle_resolution" type="double" value="6" />

  <param name="enable_pre_downsample" type="bool" value="false" />
  <param name="overlap_threshold" type="double" value="0.14" />
  
  <param name="min_time_diff" type="double" value="15.0" />
  <param name="lc_dist_threshold" type="double" value="5.0" />
  <param name="similarity_threshold" type="double" value="0.4" />

  <param name="result_save_path" type="string" value="/media/zz/new/myMidImg/search_result/kitti360_02.txt" />

  <node name="run_osk" pkg="osk" type="run_osk" output="screen" />

 <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find osk)/rviz_config/osk.rviz" />
</launch>